packages/amazonq/test/unit/amazonqGumby/transformationResultsHandler.test.ts

```diff
@@ -10,11 +10,21 @@ import path from 'path'
 import { getTestResourceFilePath } from './amazonQGumbyUtil'
 import { fs } from 'aws-core-vscode/shared'
 
+type PatchDescription = {
```

Comment:
Should go in: https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/codewhisperer/models/model.ts